### ISSUE TITLE
BardFlight def airlines(self): TypeError: 'type' object is not subscriptable

### ISSUE THREAD

**Issue by bevenets** [dsdanielpark/Bard-API#216]
💚💜 Thank you for interest. ❤️💛
Please make sure to check for more efficient package management. *Please prioritize checking existing issues first. I will repay with higher-quality code.*

**To Reproduce**
I get the stacktrace when running flask run and from bardapi import Bard
once I delete: from bardapi import Bard
and comment out all bard related stuff my project works fine. so it's only Bard.

----------Please delete the content above this line, including this line.-------------

**Describe the bug**

Keep getting the following stacktrace when I ran flask run. I'm using venv and have API key and cookies set up.

It works on another mac but not on mine but no idea how to figure out what the issue is?

Traceback (most recent call last):
  File "/Users/benitosanchez/Documents/GitHub/promise/.venv/bin/flask", line 8, in <module>
    sys.exit(main())
  File "/Users/benitosanchez/Documents/GitHub/promise/.venv/lib/python3.8/site-packages/flask/cli.py", line 1064, in main
    cli.main()
  File "/Users/benitosanchez/Documents/GitHub/promise/.venv/lib/python3.8/site-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
  File "/Users/benitosanchez/Documents/GitHub/promise/.venv/lib/python3.8/site-packages/click/core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/benitosanchez/Documents/GitHub/promise/.venv/lib/python3.8/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/benitosanchez/Documents/GitHub/promise/.venv/lib/python3.8/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
  File "/Users/benitosanchez/Documents/GitHub/promise/.venv/lib/python3.8/site-packages/click/decorators.py", line 92, in new_func
    return ctx.invoke(f, obj, *args, **kwargs)
  File "/Users/benitosanchez/Documents/GitHub/promise/.venv/lib/python3.8/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
  File "/Users/benitosanchez/Documents/GitHub/promise/.venv/lib/python3.8/site-packages/flask/cli.py", line 912, in run_command
    raise e from None
  File "/Users/benitosanchez/Documents/GitHub/promise/.venv/lib/python3.8/site-packages/flask/cli.py", line 898, in run_command
    app = info.load_app()
  File "/Users/benitosanchez/Documents/GitHub/promise/.venv/lib/python3.8/site-packages/flask/cli.py", line 313, in load_app
    app = locate_app(import_name, None, raise_if_not_found=False)
  File "/Users/benitosanchez/Documents/GitHub/promise/.venv/lib/python3.8/site-packages/flask/cli.py", line 219, in locate_app
    __import__(module_name)
  File "/Users/benitosanchez/Documents/GitHub/promise/app.py", line 6, in <module>
    from bardapi import Bard
  File "/Users/benitosanchez/Documents/GitHub/promise/.venv/lib/python3.8/site-packages/bardapi/__init__.py", line 4, in <module>
    from bardapi.core import Bard
  File "/Users/benitosanchez/Documents/GitHub/promise/.venv/lib/python3.8/site-packages/bardapi/core.py", line 28, in <module>
    from bardapi.models.result import BardResult
  File "/Users/benitosanchez/Documents/GitHub/promise/.venv/lib/python3.8/site-packages/bardapi/models/result.py", line 3, in <module>
    from bardapi.models.draft import BardDraft
  File "/Users/benitosanchez/Documents/GitHub/promise/.venv/lib/python3.8/site-packages/bardapi/models/draft.py", line 5, in <module>
    from bardapi.models.tools.flight import BardFlightContent
  File "/Users/benitosanchez/Documents/GitHub/promise/.venv/lib/python3.8/site-packages/bardapi/models/tools/flight.py", line 4, in <module>
    class BardFlight:
  File "/Users/benitosanchez/Documents/GitHub/promise/.venv/lib/python3.8/site-packages/bardapi/models/tools/flight.py", line 17, in BardFlight
    def airlines(self) -> list[str]:
TypeError: 'type' object is not subscriptable
**Version**
OS:
Python:
Bard API:
Using proxy:
Legion:

**Code**
```python
code line
```

**Error**
```
error message
```

**Comment by memocampos** [comment]
Same issue, in my case it failed with AWS Beanstalk here is the error:

Oct 23 18:30:39 ip-172-31-13-230 web: Traceback (most recent call last):
Oct 23 18:30:39 ip-172-31-13-230 web: File "/var/app/venv/staging-LQM1lest/lib64/python3.8/site-packages/gunicorn/arbiter.py", line 589, in spawn_worker
Oct 23 18:30:39 ip-172-31-13-230 web: worker.init_process()
Oct 23 18:30:39 ip-172-31-13-230 web: File "/var/app/venv/staging-LQM1lest/lib64/python3.8/site-packages/gunicorn/workers/gthread.py", line 92, in init_process
Oct 23 18:30:39 ip-172-31-13-230 web: super().init_process()
Oct 23 18:30:39 ip-172-31-13-230 web: File "/var/app/venv/staging-LQM1lest/lib64/python3.8/site-packages/gunicorn/workers/base.py", line 134, in init_process
Oct 23 18:30:39 ip-172-31-13-230 web: self.load_wsgi()
Oct 23 18:30:39 ip-172-31-13-230 web: File "/var/app/venv/staging-LQM1lest/lib64/python3.8/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
Oct 23 18:30:39 ip-172-31-13-230 web: self.wsgi = self.app.wsgi()
Oct 23 18:30:39 ip-172-31-13-230 web: File "/var/app/venv/staging-LQM1lest/lib64/python3.8/site-packages/gunicorn/app/base.py", line 67, in wsgi
Oct 23 18:30:39 ip-172-31-13-230 web: self.callable = self.load()
Oct 23 18:30:39 ip-172-31-13-230 web: File "/var/app/venv/staging-LQM1lest/lib64/python3.8/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
Oct 23 18:30:39 ip-172-31-13-230 web: return self.load_wsgiapp()
Oct 23 18:30:39 ip-172-31-13-230 web: File "/var/app/venv/staging-LQM1lest/lib64/python3.8/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
Oct 23 18:30:39 ip-172-31-13-230 web: return util.import_app(self.app_uri)
Oct 23 18:30:39 ip-172-31-13-230 web: File "/var/app/venv/staging-LQM1lest/lib64/python3.8/site-packages/gunicorn/util.py", line 359, in import_app
Oct 23 18:30:39 ip-172-31-13-230 web: mod = importlib.import_module(module)
Oct 23 18:30:39 ip-172-31-13-230 web: File "/usr/lib64/python3.8/importlib/__init__.py", line 127, in import_module
Oct 23 18:30:39 ip-172-31-13-230 web: return _bootstrap._gcd_import(name[level:], package, level)
Oct 23 18:30:39 ip-172-31-13-230 web: File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
Oct 23 18:30:39 ip-172-31-13-230 web: File "<frozen importlib._bootstrap>", line 991, in _find_and_load
Oct 23 18:30:39 ip-172-31-13-230 web: File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
Oct 23 18:30:39 ip-172-31-13-230 web: File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
Oct 23 18:30:39 ip-172-31-13-230 web: File "<frozen importlib._bootstrap_external>", line 843, in exec_module
Oct 23 18:30:39 ip-172-31-13-230 web: File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
Oct 23 18:30:39 ip-172-31-13-230 web: File "/var/app/current/application.py", line 3, in <module>
Oct 23 18:30:39 ip-172-31-13-230 web: from bardapi import Bard
Oct 23 18:30:39 ip-172-31-13-230 web: File "/var/app/venv/staging-LQM1lest/lib64/python3.8/site-packages/bardapi/__init__.py", line 4, in <module>
Oct 23 18:30:39 ip-172-31-13-230 web: from bardapi.core import Bard
Oct 23 18:30:39 ip-172-31-13-230 web: File "/var/app/venv/staging-LQM1lest/lib64/python3.8/site-packages/bardapi/core.py", line 29, in <module>
Oct 23 18:30:39 ip-172-31-13-230 web: from bardapi.models.result import BardResult
Oct 23 18:30:39 ip-172-31-13-230 web: File "/var/app/venv/staging-LQM1lest/lib64/python3.8/site-packages/bardapi/models/result.py", line 3, in <module>
Oct 23 18:30:39 ip-172-31-13-230 web: from bardapi.models.draft import BardDraft
Oct 23 18:30:39 ip-172-31-13-230 web: File "/var/app/venv/staging-LQM1lest/lib64/python3.8/site-packages/bardapi/models/draft.py", line 5, in <module>
Oct 23 18:30:39 ip-172-31-13-230 web: from bardapi.models.tools.flight import BardFlightContent
Oct 23 18:30:39 ip-172-31-13-230 web: File "/var/app/venv/staging-LQM1lest/lib64/python3.8/site-packages/bardapi/models/tools/flight.py", line 4, in <module>
Oct 23 18:30:39 ip-172-31-13-230 web: class BardFlight:
Oct 23 18:30:39 ip-172-31-13-230 web: File "/var/app/venv/staging-LQM1lest/lib64/python3.8/site-packages/bardapi/models/tools/flight.py", line 17, in BardFlight
Oct 23 18:30:39 ip-172-31-13-230 web: def airlines(self) -> list[str]:
Oct 23 18:30:39 ip-172-31-13-230 web: TypeError: 'type' object is not subscriptable

**Comment by dsdanielpark** [comment]
It will be updated to the official version after addressing errors in some type hints and conducting QA on the overall structure. Until then, we appreciate your guidance on any errors encountered after installing the GitHub version.

**Comment by matheus-giordani** [comment]
same problem but my code is very more simple 

```from bardapi import Bard

import os 
os.environ["_BARD_API_KEY"] = "my_api_key"
print(Bard.get_answer(" 4 + 4"))

// error:

Traceback (most recent call last):
  File "teste.py", line 1, in <module>
    from bardapi import Bard
  File "/home/matheus_giordani/.local/lib/python3.8/site-packages/bardapi/__init__.py", line 4, in <module>
    from bardapi.core import Bard
  File "/home/matheus_giordani/.local/lib/python3.8/site-packages/bardapi/core.py", line 28, in <module>
    from bardapi.models.result import BardResult
  File "/home/matheus_giordani/.local/lib/python3.8/site-packages/bardapi/models/result.py", line 3, in <module>
    from bardapi.models.draft import BardDraft
  File "/home/matheus_giordani/.local/lib/python3.8/site-packages/bardapi/models/draft.py", line 5, in <module>
    from bardapi.models.tools.flight import BardFlightContent
  File "/home/matheus_giordani/.local/lib/python3.8/site-packages/bardapi/models/tools/flight.py", line 4, in <module>
    class BardFlight:
  File "/home/matheus_giordani/.local/lib/python3.8/site-packages/bardapi/models/tools/flight.py", line 17, in BardFlight
    def airlines(self) -> list[str]:
TypeError: 'type' object is not subscriptable

**Comment by dsdanielpark** [comment]
23-10-28, switched to using the typing module for providing type hints by importing it.

- The notation like `list[str]` is related to type hints available in Python 3.9 and later. However, according to the error message, it appears that this notation is not supported.
